tensorflow/python/ops/image_ops_impl.py

```diff
@@ -2931,13 +2932,21 @@ def rgb_to_yiq(images):
   Outputs a tensor of the same shape as the `images` tensor, containing the YIQ
   value of the pixels.
   The output is only well defined if the value in images are in [0,1].
+  
+  Usage Example:
+    ```python
+    >>> x = tf.constant([[[1.0, 2.0, 3.0]]])
+    >>> tf.image.rgb_to_yiq(x)
+    <tf.Tensor: shape=(1, 1, 3), dtype=float32, numpy=array([[[ 1.815     , -0.9...,  0.09...]]], dtype=float32)>
```

Comment:
The floats will be rounded off to 1e-6 automatically. For that reason, `...` in floats is not allowed.

So you can copy-paste the output directly and it should start working.

Reply:
Also you have ``` at the end, causing the test to fail.